internal/handlers/telegram/documentHandler.go

```diff
@@ -0,0 +1,39 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+)
+
+
+type DocumentObject struct {
+	EnableLink bool
+	Link string
+	Object *tgbotapi.Document
+}
+
+/*
+Constructs a new document and is used to decide under which format to formard
```

Comment:
This relates to the issue we have up on how to do proper development documentation. GoDoc uses comments to generate dev docs. Start function comments with "functionName does...."

In this case, we can start off with "newDocument constructs a new document and is..."